test/gtest/ucp/test_ucp_tag_offload.cc

```diff
@@ -550,19 +550,20 @@ UCS_TEST_P(test_ucp_tag_offload_multi, recv_from_multi)
 {
     ucp_tag_t tag = 0x11;
 
-    // Activate first offload iface. Tag hashing is not done yet, since we
-    // have only one active iface so far.
+    // Activate first offload iface. Tag hashing is performed only if there is
+    // more than one active interface.
     activate_offload_hashing(e(0), make_tag(e(0), tag));
-    EXPECT_EQ(0u, kh_size(&receiver().worker()->tm.offload.tag_hash));
+    int init_hash_size = receiver().worker()->num_active_ifaces > 1;
```

Comment:
Do I understand correctly that no matter how many `num_active_ifaces` do we have here, if there are more than 1 of them, `tag_hash` size should be equal to 1?

Reply:
> after this change, the test probably does not need to try limiting number of lanes to 1, or need to add some comment

updated in place (PR is small)